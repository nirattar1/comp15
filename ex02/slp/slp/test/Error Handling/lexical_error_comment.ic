class Stam{
	int run(){

		/* If you don't close the comment,
		Your code won't work


		for (i=0; i<100;i+=1){
		// even if makes a lot of sense
		print(i);
		}
	}
}
