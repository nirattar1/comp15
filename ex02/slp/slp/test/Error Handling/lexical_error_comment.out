14: Lexical error: Unexpected end of file